src/docker/mod.rs

```diff
@@ -1,8 +1,10 @@
 mod engine;
 mod local;
+pub mod remote;
 mod shared;
 
 pub use self::engine::*;
+pub use self::remote::*;
```

Comment:
I think glob importing should be used when it's a prelude or alike, not for this